src/clientdlg.cpp

```diff
@@ -333,6 +333,9 @@ CClientDlg::CClientDlg ( CClient*         pNCliP,
     QAction* ByCityAction =
         pViewMenu->addAction ( tr ( "Sort Users by &City" ), this, SLOT ( OnSortChannelsByCity() ), QKeySequence ( Qt::CTRL + Qt::Key_T ) );
 
+    QAction* ByChannelAction =
+        pViewMenu->addAction ( tr ( "Sort Users by Ch&annel" ), this, SLOT ( OnSortChannelsByChannel() ), QKeySequence ( Qt::CTRL + Qt::Key_A ) );
```

Comment:
Maybe this is not expressive enough and too technical. 
We should make sure that the user knows it's mostly "something todo with MIDI" and not random.

Reply:
@gilgongo Any ideas to make this concise but not too technical?
The ordering seems "random" unless midi is enabled.

Reply:
Until now, we've always used "Channel" to mean the audio signal as part of a mix - so does this introduce a new meaning of "MIDI channel"? 

If so, I think we'd need to say, "Sort Users by MIDI Channel" (although I'd prefer we said just "Sort by MIDI Channel" really, because it should be obvious that you're sorting the UI Faders/Users).

Reply:
No, this is nothing to do with MIDI Channel.  This is the Jamulus Server-assigned Channel ID.  It's inherent to how `--ctrlmidich` works, hence my statement in the related issue that the documentation needs to make the meaning of the menu option clear.

Reply:
Oh OK. "Sort by Channel" does feel a bit elliptical somehow (like a numeric list being "sorted by value"). Would "Sort by assigned Channel" be any clearer, assuming the curious would then read the docs on it? An alternative could be "Default sort", which again would be expanded upon in the docs.

Reply:
It's not the default sort, as explained above.  The default sort, explicitly, puts new channels on the left of the mixer - that's a definition and it's intended to work that way so that the existing channels, when default sort is selected, don't keep moving around.  Someone leaves, their fader is removed and everyone moves left.  Someone joins, they're added at the right-hand end.  I don't think that should be changed.

"Assigned" begs the question of "by whom?".  As the `--ctrlmidich` docs answer that -- it's the Server -- then can I suggest "Sort by Server Channel"?

Reply:
By the way, I've added a change to the `--ctrlmidich` Pull Request that would mean the "Sort by Server Channel" could either be invisible (which would potentially be confusing) or disabled if MIDI controls were not enabled.  I'm not convinced it would help anyone, though.

Reply:
"Sort by Server Channel" is perhaps marginally less "recursive" from the POV of someone who knows nothing about `--ctrlmidich`. Although I'd still expect somebody who was curious to simply look it up in the docs, because how the Server applies the sort would otherwise be mysterious to them. 

I don't think there's an easy way of wording it :-)

Reply:
Sort by Server channel is ok I think.